project-standards/technical-agility/up-to-date-dependencies.s.md

```diff
@@ -19,3 +19,26 @@
 
 ## Good Examples
 *TBD*
+
+## Upgrading React Native
+
+React Native is very probably the dependency you'll have to upgrade the more often so we felt it was worth dedicating a whole section to it.
+
+### How to
+
+First of all, if you are late by several versions do them one at a time.
+Skipping one might be really tough and you can get stuff done more often if you do it one version at a time.
+
+To upgrade a version follow [the official documentation](https://facebook.github.io/react-native/docs/upgrading.html) on how to use `react-native-git-upgrade`.
```

Comment:
Je dirais plutôt :

- "Before upgrading, check that you project dependencies are compatible with your aimed version of React Native, like for Codepush, that always pushes a new version, for new RN versions"